### PR TITLE
Escape special characters in module lookup error

### PR DESCRIPTION
**Pull request by TheDarkSid3r**

### CODE REVIEW

**Comment by TheDarkSid3r** [comment]
Things like this can happen:
image

While not critical, it seems like something that _should_ be fixed.

**Comment by Qkrisi** [comment]
Doesn't really seem like it fixed that issue.

image

**Comment by TheDarkSid3r** [comment]
ohh right that's my bad. hold on

**Comment by TheDarkSid3r** [comment]
(Removed the code that escaped the other characters too; I forgot that all characters are technically fine when inside backticks. Except backticks.)

**Comment by Cyber28** [comment]
This solution is way too hacky and unconventional

File: src/utils.js [hunk 1]
```diff
@@ -76,7 +76,8 @@ exports.GetModule = (message, args, send = true) => {
 			result = GetMatching(args._.join(' '))
 			if (result && result.length > 0) return HandleRegex(result)
 			else {
-				if (send) message.channel.send(`🚫 Couldn't find a module by the ID of \`${args._[0]}\` (case-sensitive), name of \`${args._.join(' ')}\` (not case-sensitive) or periodic symbol of \`${args._[0]}\` (not case-sensitive)`)
+				const cleanseDiscordText = (text) => text.replace(/`/g, "");
```
Comment (Cyber28) [inline src/utils.js:79]: This shouldn't be defined every time the `GetModule` function is called

**Review by Cyber28 — CHANGES_REQUESTED** [review]

File: src/utils.js (hunk 1 above)
Comment (TheDarkSid3r) [reply src/utils.js:79]: Understandable

**Pull request merged**